### DISCUSSION TITLE
Extending log retention of a hosted Supabase project

### DISCUSSION THREAD

**Discussion by GotHima** [supabase/supabase#19897] [Questions]
Based on this article (https://supabase.com/docs/reference/self-hosting-analytics/introduction) we are able to self-host a Logflare instance and subsequently configure the BigQuery backend ourselves.

Is this only applicable to the scenario where we want to self-host the whole Supabase instance? Or can we do the same with a hosted Supabase project (i.e. on supabase.com)? It would be even better if we can directly configure a Supabase project to use our own BigQuery backend.

The reason I'm asking is this is because I would like to find a way to store the log data for some Supabase tools (API gateway, Postgres databases, Storage, Edge Functions, Auth, and Realtime) beyond the log retention imposed in each Supabase subscription plan.

Thanks!

**Comment by Ziinc** [comment]
It is currently not possible, beyond enterprise-level support. 
We are currently working on a solution which would enable this possibility, do stay tuned.
